packages/paste-website/src/pages/content/product-style-guide.mdx

```diff
@@ -192,56 +200,118 @@ Coming in the next release
+  - *I understand.* (This usage is almost exclusively for legal consent.)
+
+<Callout>
+  <CalloutTitle as="h4">Legal content</CalloutTitle>
+  <CalloutText>Legal content follows different rules to product &mdash; e.g., in Terms of Service, Twilio is usually referenced in the third person. Always check with the legal team if the content regards consent. If they suggest you use specific wording, <strong>use the wording the legal team suggests</strong>. 
+  </CalloutText>
+</Callout>
+
+**Do** say *we* instead of *Twilio*
+- It's us talking, so there's no need to refer to us in the third person.
+- If "Twilio" is included in a product or feature name, that's ok as that's using it as a noun.
 
 #### Lists (all)
 **Do** start each list item with the same part of speech – usually a noun or verb – to make the list "parallel."
 - Punctuate the end of each item with a period if it is a complete sentence.
 - Be consistent within the list. If one item takes a full point, all items should.
 
-**Don't** punctuate the end of an item in a vertical list with a comma or elipsis "..."
+**Don't** punctuate the end of an item in a vertical list with a comma or ellipsis "..."
 
 #### Lists (bullet points)
-- **Do** use bulleted lists for items linked by a topic. Make the list topic clear in the sentence or heading that precedes it.
+**Do** use bulleted lists for items linked by a topic. Make the list topic clear in the sentence or heading that precedes it.
 
 #### Lists (numbered)
-- **Do** use numbered lists for items to be done or understood in a sequence.
-- **Avoid** listing more than 7 items.
+**Do** use numbered lists for items to be done or understood in a sequence.
+
+**Avoid** listing more than 7 items &mdash; it's a lot to take in. Break a big lists into smaller ones with explanatory headings.
 
 #### Lists (sentence)
 **Do** use a sentence list if you’re explaining a short process, noting up to four options, or highlighting up to four key points. You just read a sentence list.
 - Sentence lists should be used for no more than four items. Over four? List with bullet points.
 - Use the Oxford comma before the last item in a sentence list.
 
 #### Numbers
-Coming in the next release
+**Do** use precise numbers to express statistics, metrics, or quantities. Avoid rounding up or down when the user is depending on the information to be accurate. 
+
+**Do** use numerals (1, 2, 3, 45, 101) instead of spelling out the number.
+- Percentages *49%*
+- Ratios *3 to 1*
+
+**Do** write expressions and ordinal numbers in words.
+- *One to watch, thousands of options*
+- *First, create a profile*
+
+**Do** use both words and numerals to represent numbers over 1000, unless the user needs a specific number. 
+- *Up to 40k emails*
+- *More than $1.5B*
+- **Note** this is most commonly used in marketing content &mdash make sure you're giving the user what they need to complete a task, and not overloading them with extra info. 
```

Comment:
I think you should be able to use "—" here instead of writing out "&mdash". If not, "&mdash" here is missing a semicolon.

Reply:
TY